src/Containers/Info/index.js

```diff
@@ -0,0 +1,378 @@
+              </Link>
+              . A motorized electronic system creates movement from within the
+              fabric, whose shape and color is designed in conjunction with its
+              gestures. They are meant to be interacted with, as people’s
+              reactions to these animate objects are key to Ataman’s practice.
+              Again, a synthetic system is designed to induce affect. Underwater
+              animals, poison, and the emotional pull of a silent but sentient
+              creature are all called to mind, layers of stimuli to invite
+              perception and its emotional effects.
+            </p>
+            <p>
+              Continuing this vein of inquiry, Leming Chung presents{" "}
+              <Link to="/leming-zc" className={infoLink}>
+                My doctor's prescription for my pollen allergy is to let the
+                light illuminate everywhere
+              </Link>{" "}
+              the video element of a larger installation stemming from her
+              interest in biopolitics and networks. The uneasy division between
+              the self and the other, what belongs and what does not, is
+              narrated through the story of a single bee who experiences an
+              allergic reaction. Like the allergen itself, this bee is an
+              innocuous part of a whole turned into an invader. Unlike say, a
+              virus, an allergy is an internal process that consists of the
+              body’s overreaction to an otherwise harmless event. Taking these
+              minuscule life forms as starting points, Chung encourages
+              reflection on the diseased and disrupted individual body, as well
+              the social body’s efforts to control it.
+            </p>
+            <p>
+              The artists negotiate movement through landscapes and networks,
+              among the human, non-human, and the in-between. This with a
+              recognition of nearness, the gravity of our intimacies, calculated
+              proximity and distance in computation, within a cohort, a socially
+              distant society, an exhibition.n.
+            </p>
+            <p>
+              <b>-Ana Iwataki</b>
+            </p>
+            <p className={citeText}>
+              <Link className={citeLink}>
+                <sup>[1]</sup>
+              </Link>{" "}
+              Neimanis, Astrida. “On Collaboration (for Barbara Godard).” NORA -
+              Nordic Journal of Feminist and Gender Research, vol. 20, no. 3,
+              2012, pp. 215–221., doi:10.1080/08038740.2012.703689.
+            </p>
+          </div>
+        </Column>
+      </ProjectColumns>
+      <ProjectColumns className={infoWrapper}>
+        <Column>
+          <h2 className={intoTitle}>
+            The New Dematerialization of Art under “Covid-19 Rule”
+          </h2>
+          <h3>A Response from UCLA Design Media Arts Department</h3>
```

Comment:
instead of making these `<h3>`, accessibility-wise these would be better placed in the same `<h2>` block using a `<span className="blah">` to style differently

Reply:
i'll make these changes